docs/configuration/hyperparameter_optimization.md

```diff
@@ -186,7 +186,7 @@ The `ray` executor is used to enable [Ray Tune](https://docs.ray.io/en/master/tu
 - `kubernetes_namespace`: When running on Kubernetes, provide the namespace of the Ray cluster to sync results between pods. See the [Ray docs](https://docs.ray.io/en/master/_modules/ray/tune/integration/kubernetes.html) for more info.
 - `time_budget_s`: The number of seconds for the entire hyperopt run.
 - `max_concurrent_trials`: The maximum number of trials to train concurrently. Defaults to `auto` if not specified.
-  - If you're using a Ray backend, `auto` will infer the max_concurrent_trials that can be set given your cluster configuration.
+  - If you're using a Ray backend, `auto` will infer the max_concurrent_trials that can be set given your cluster configuration to prevent trials from stalling.
   - If you're using a Local or Horovod backend, `auto` will set max_concurrent_trials to None.
```

Comment:
Strange that it doesn't seem like anything has changed in the changelog

Reply:
Ok I see what's happening. Pre-commit basically nests this back one level which is causing problems. So this isn't working. Any suggestions on how else to make nested lists @justinxzhao ?

Reply:
I could disable pre-commit and make this change, but it might cause problems when other people work on these files in the future.

Reply:
It does seem odd why pre-commit pins the sub-bullets to a 2 space indent.

As a workaround for now, I'm stuffed sub-bullets in an admonition.

Reply:
<img width="784" alt="image" src="https://user-images.githubusercontent.com/3459541/196744197-95ff63b4-d4fe-4afa-a445-9c7439fe0588.png">

Reply:
Nice, thanks!